<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductOutDialog</class>
 <widget class="QDialog" name="ProductOutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>703</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>703</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EasyManager-ProductOut</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListView
{
font:bold 15px;
}
QLabel
{
font:bold 15px;
color:white
}
QDialog
{
background-image:url(&quot;:/Theme/BackImage/DialogBack_7.jpg&quot;)
}
QPushButton
{
font:bold 15px;

}</string>
  </property>
  <widget class="QLineEdit" name="ProductChooseEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumberBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择的商品：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出库数量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ComfirmButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出库单价</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CostEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ProductView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前存量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="CountLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ProductViewLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
